docs/toc.json

```diff
@@ -67,6 +67,21 @@
   },
   "examples": {
     "name": "Examples",
+    "children": {
+      "examples_basic": "Basic",
+      "examples_advanced": "Advanced"
```

Comment:
These don't seem to be children of this chapter. @lucacasonato Will this work?

Reply:
Oh nevermind, this will be addressed with the previous review.

Reply:
I'm going to change it back to just examples as @ry requested.